src/aurora/actionscript/object.cpp

```diff
@@ -80,6 +80,11 @@ Variable Object::call(Common::UString function, AVM &avm) {
 		counter += 1;
 	}
 
+	for (int i = 0; i < arguments.size(); ++i) {
```

Comment:
`error: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<Aurora::ActionScript::Variable>::size_type’ {aka ‘long unsigned int’} [-Werror=sign-compare]`

Make that `i` a `size_t`, please.